c2corg_ui/templates/index.html

```diff
@@ -38,4 +38,20 @@ <h1 class="welcome-title text-center" translate>The mountain sports community</h
     </div>
   </section>
   <app-feed id="feed"></app-feed>
+
+  <div class="float-buttons" ng-if="userCtrl.auth.isAuthenticated()">
+    <div class="float-button-container">
+      <div class="float-button" ng-click="feedCtrl.toggleFilters()" ng-class="{'personal-feed-on': feedCtrl.isPersonal}">
+        <span class="glyphicon glyphicon-user"></span>
+      </div>
+      <p class="float-button-text" ng-bind="feedCtrl.isPersonal ? ('Personal feed on' | translate) :  ('Personal feed off' | translate)">Personal feed</p>
```

Comment:
it's a "dummy" text so that there is anything while the pages loads. I think I should to set it to "personal feed off" by default.